src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int firstLeg;
+    private int secondLeg;
+
+    public RightTriangle(String color, int firstLeg, int secondLeg) {
+        this.color = color;
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double getArea() {
+        return firstLeg * secondLeg * 0.5;
```

Comment:
The multiplication by 0.5 is a magic number. It would be better to define it as a constant with a descriptive name, such as `HALF`, to improve readability and maintainability of your code.